src/raftstore/store/peer_storage.rs

```diff
@@ -791,6 +789,24 @@ impl PeerStorage {
         self.cache.compact_to(idx);
     }
 
+    pub fn maybe_gc_cache(&mut self, replicated_idx: u64, apply_idx: u64) {
+        if replicated_idx == apply_idx {
```

Comment:
Should be `>=`?

Reply:
Seems both are ok. But I think `>=` is easier to understand.

Reply:
There is compaction rolling all the time, so this strategy is just a supplement for the abnormal cases. One is if the region is inactive, there may be also some logs not compacted, but they are kept in memory. That's the reason why checking equality here. The other is isolation. In this case, `replicated_idx` is not reliable. For example, all follower's progress of a just elected leader will become 0. Just checking equality can reduce the bad decision.